src/chart/rendering/painter/coordinatesystem.cpp

```diff
@@ -98,12 +98,12 @@ Math::FuzzyBool PolarDescartesTransform::getPolar() const
 
 bool PolarDescartesTransform::atEndState() const
 {
-	return polar == 0 || polar == 1;
+	return polar == false || polar == true;
```

Comment:
Older implementations of Fuzzybool would had been thrown here for any non-integer value of polar, this is why this was 0, 1 instead of true, false. Does this safe (non-throwing) with the current implementation?

Reply:
For FuzzyBool, only the bool equality operator is implemented (no other == operators). As a result, when using this class, 0 and 1 are treated as bool values rather than being compared as int/double.